.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: python
+language: bash
```

Comment:
What does this field mean and why was it changed?

Reply:
Because `travis` doesn't support `Python` as supported language for `Windows`. 

More information [here](https://docs.travis-ci.com/user/reference/windows/#supported-languages)